data_ingestion.ipynb

```diff
@@ -22,7 +22,7 @@
         "colab_type": "text"
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #23.    else:
</pre><p>If you want, and if it works, I think it would be fine to add a pointer to the exploratory partitioning strategy you came up with, as long as there's a big warning that it's for research / helping define an initial fixed partitioning strategy for a given dataset, but not necessarily stable between runs (?) or for long-term use as we add data.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tnc-br/ddf-isoscapes/pull/191/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>